config.go

```diff
@@ -366,6 +367,22 @@ func loadConfig() (*config, []string, error) {
 		return nil, nil, err
 	}
 
+	// Exit if you try to use a simulation wallet with a standard
+	// data directory.
+	if cfg.DataDir == defaultDataDir && cfg.CreateTemp {
+		fmt.Fprintln(os.Stderr, "Tried to create a temporary simulation "+
+			"wallet, but failed to specify data directory!")
+		os.Exit(0)
+	}
+
+	// Exit if you try to use a simulation wallet on anything other than
+	// simnet or testnet3.
+	if (!cfg.SimNet || cfg.MainNet) && cfg.CreateTemp {
```

Comment:
I think this just be `if !cfg.SimNet && cfg.CreateTemp {`.  The code above has already verified that only a single network is selected.  Also, the comment above should just say 'other than simnet.'